packages/office-ui-fabric-react/src/components/ThemeGenerator/ThemeGeneratorPage.tsx

```diff
@@ -121,6 +115,37 @@ export class ThemeGeneratorPage extends BaseComponent<{}, IThemeGeneratorPageSta
           * and generate a theme based off of those.
           * Since this API requires a personal subscription key, you'll have to enlist and insert your subscription
           * key in _makeThemeFromImg() below. Then, just uncomment this section. */}
+        {/*
+          private _imgUrl: string;
+          const BackgroundImageUriKey = 'backgroundImageUri';
+          const BackgroundOverlayKey = 'backgroundOverlay';
+
+          private _makeThemeFromImg = (): void => {
+            this._imgUrl = (document.getElementById('imageUrl') as HTMLInputElement).value;
+            (document.getElementById('imagePreview') as HTMLImageElement).src = this._imgUrl;
+
+            if (this._imgUrl) {
+              const xhr = new XMLHttpRequest();
+              xhr.addEventListener('load', this._cognitiveVisionCallback.bind(this));
+              // you may need to change the URL here
+              xhr.open('POST', 'https://westus.api.cognitive.microsoft.com/vision/v1.0/analyze?visualFeatures=Description%2CColor&details=&language=en');
+              xhr.setRequestHeader('Content-Type', 'application/json');
+              alert('You forgot to set the subscription key!');
+              xhr.setRequestHeader('Ocp-Apim-Subscription-Key', 'YourSubscriptionKeyHere'); // put your subscription key here
+              xhr.send('{ "url": "' + this._imgUrl + '" }');
+            } else {
+              // remove related properties from theme
+              const { themeRules } = this.state;
+              if (themeRules.hasOwnProperty(BackgroundImageUriKey)) {
+                delete themeRules[BackgroundImageUriKey];
+              }
+              if (themeRules.hasOwnProperty(BackgroundOverlayKey)) {
+                delete themeRules[BackgroundOverlayKey];
+              }
+              this.setState({ themeRules: themeRules }, this._makeNewTheme);
+            }
+          }
+        */}
```

Comment:
What's the reason for keeping this? (It looks like `_cognitiveVisionCallback` was removed in the PR but is still called in the commented out code.)